docs/quick-answers/linode-platform/how-to-change-your-email/index.md

```diff
@@ -0,0 +1,134 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'This quick answer guide shows you how to change your account email address both in the Linode Cloud Manager and by using the Linode API.'
+og_description: 'This quick answer guide shows you how to change your account email address both in the Linode Cloud Manager and by using the Linode API.'
+keywords: ["email address", "email addresses", "address", "addresses"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-01-20
+modified_by:
+  name: Linode
+published: 2020-01-20
+title: How to Change Your Email Address
+h1_title: Changing Your Email Address
+---
+
+Changing your account's email address is easy and can be done two different ways.
+
+## From the Cloud Manager
+
+1.  Log in to the [Cloud Manager](http://cloud.linode.com).
+
+1.  Click on your username and choose **My Profile** from the drop down menu.
+
+1.  On the My Profile page, in the **Email** field, enter the email you would like to associate with your account.
+
+    ![Click My Profile from Username Drop Down Menu](how-to-change-email-my-profile.png "Click My Profile from Username Drop Down Menu")
+
+1.  Click the **Save** button.
+
+1.  Cloud Manager will tell you that the email address has been updated. You will also receive an email at this new address from Linode Support confirming this update.
+
+    ![Click to Save a New Email](how-to-change-email-save.png "Click to Save a New Email")
```

Comment:
I think this image should be moved up to step 3.

Reply:
I moved this to under the Save button